manim/default.cfg

```diff
@@ -0,0 +1,144 @@
+# manim.cfg
+# Default configuration for manim
+
+# Configure how manim behaves when called from the command line without
+# specifying any flags
+[CLI]
+
+# Each of the following will be set to True if the corresponding CLI flag
+# is present when executing manim.  If the flag is not present, they will
+# be set to the value found here.  For example, running manim with the -w
+# flag will set WRITE_TO_MOVIE to True.  However, since the default value
+# of WRITE_TO_MOVIE defined in this file is also True, running manim
+# without the -w value will also output a movie file.  To change that, set
+# WRITE_TO_MOVIE = False so that running manim without the -w flag will not
+# generate a movie file.  Note all of the following accept boolean values.
+
+# -w, --write_to_movie
+write_to_movie = True
+
+# -a, --write_all
+write_all = False
+
+# -s, --save_last_frame
+save_last_frame = False
+
+# -g, --save_pngs
+save_pngs = False
+
+# -i, --save_as_gif
+save_as_gif = False
+
+# -p, --preview
+preview = False
+
+# -f, --show_file_in_finder
+show_file_in_finder = False
+
+# -q, --quiet
+quiet = False
+
+# --sound
+sound = False
+
+# -o, --output_file
+output_file =
+
+# --leave_progress_bars
+leave_progress_bars = False
+
+# -c, --color
+background_color = BLACK
+
+# --background_opacity
+background_opacity = 1
+
+# The following two are both set by the -n (or --from_animation_number)
+# flag.  See manim -h for more information.
+from_animation_number = 0
+
+# Use -1 to render all animations
+upto_animation_number = -1
+
+# The following are meant to be paths relative to the point of execution.
+# Set them at the CLI with the corresponding flag, or set their default
+# values here.
+# --media_dir
+media_dir = ./media
+
+# --video_dir
+video_dir = %(MEDIA_DIR)s/videos
+
+# --tex_dir
+tex_dir = %(MEDIA_DIR)s/Tex
+
+# --text_dir
+text_dir = %(MEDIA_DIR)s/texts
+
+# This value cannot be changed through CLI
+gif_file_extension = .gif
```

Comment:
Not related to this PR, but is there any reason this should be configurable?

Reply:
No, but I thought it was more elegant to have it be part of the `config` dict than to hardcode it elsewhere.

Reply:
If it shouldn't configurable it'd make more sense to put it in constants.py.

Reply:
Fair. Change incoming.